SaintCoinach/SaintCoinach.csproj

```diff
@@ -2555,6 +2555,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
     <Content Include="Definitions\GatheringLeveRule.json">
```

Comment:
This tag isn't closed, it should also have a CopyToOutputDirectory directive - just copy it from another entry